torch/distributed/fsdp/_fully_shard/_fully_shard.py

```diff
@@ -350,6 +350,35 @@ def set_requires_all_reduce(
                 if fsdp_param_group := state._fsdp_param_group:
                     fsdp_param_group.all_reduce_grads = requires_all_reduce
 
+    def set_reshard_after_forward(
+        self, reshard_after_forward: bool, recurse: bool = True
+    ) -> None:
+        """
+        Sets if the module should reshard parameters after forward. This can be
+        used to change the ``_reshard_after_forward`` FSDP arg at runtime. For
```

Comment:
This is a public arg:
https://github.com/pytorch/pytorch/blob/c41c2130be9d8f8617ae92252d2b6a5932ad6899/torch/distributed/fsdp/_fully_shard/_fully_shard.py#L89
```suggestion
        used to change the ``reshard_after_forward`` FSDP arg at runtime. For
```